internal/proxy/task_search.go

```diff
@@ -328,6 +327,8 @@ func (t *searchTask) PreExecute(ctx context.Context) error {
 			return err
 		}
 
+		t.request.OutputFields = append(t.request.OutputFields, common.TimeStampFieldName)
+		outputFieldIDs = append(outputFieldIDs, common.TimeStampField)
 		t.SearchRequest.OutputFieldsId = outputFieldIDs
 		plan.OutputFieldIds = outputFieldIDs
 
```

Comment:
Changes in wrong function, change task_query.go instead

Reply:
ok